runtime/onert/backend/cpu/ops/OperationUtils.cc

```diff
@@ -308,6 +310,17 @@ std::vector<int32_t> getReducerAxes(const IPortableTensor *axes)
   return ret;
 }
 
+bool try_assign_buffer(const IPortableTensor *source, IPortableTensor *target)
+{
+  auto assignable_target_tensor = dynamic_cast<basic::Tensor *>(target);
+  if (nullptr == assignable_target_tensor)
+  {
+    return false;
+  }
+  assignable_target_tensor->setBuffer(source->buffer());
```

Comment:
I don't know or remember the buffer management in onert. Is it safe to set raw buffer of source to target? In other words, what happens when source's buffer is destroyed?

Reply:
I also think it is dangerous. In this case, memory manager cannot know buffer usage, and it may break buffer liveness.

Example

```cpp
TEST_F(GenModelTest, reshaping_and_arith)
{
  CircleGen cgen;
  auto f32 = circle::TensorType::TensorType_FLOAT32;
  int in2 = cgen.addTensor({{1, 1}, f32});
  int out3 = cgen.addTensor({{1, 1}, f32});
  cgen.addOperatorAdd({{in2, in2}, {out3}}, circle::ActivationFunctionType_NONE);

  int in1 = cgen.addTensor({{2, 2}, f32});
  int out1 = cgen.addTensor({{2, 2}, f32});
  cgen.addOperatorAdd({{in1, in1}, {out1}}, circle::ActivationFunctionType_NONE);

  int out2 = cgen.addTensor({{1, 4}, f32});
  std::vector<int32_t> shape_data = {1, 4};
  uint32_t shape_buf = cgen.addBuffer(shape_data);
  int shape = cgen.addTensor({{2}, circle::TensorType::TensorType_INT32, shape_buf});
  cgen.addOperatorReshape({{out1, shape}, {out2}});

  int out4 = cgen.addTensor({{1, 4}, f32});
  cgen.addOperatorAdd({{out2, out3}, {out4}}, circle::ActivationFunctionType_NONE);
  cgen.setInputsAndOutputs({in1, in2}, {out4});

  _context = std::make_unique<GenModelTestContext>(cgen.finish());
  _context->addTestCase(uniformTCD<float>({{1, 2, 3, 4}, {4}}, {{10, 12, 14, 16}}));
  _context->setBackends({"cpu"});

  SUCCEED();
}
```